packages/scandipwa/src/component/ProductTabs/ProductTabs.component.tsx

```diff
@@ -48,6 +60,16 @@ export class ProductTabsComponent extends PureComponent<ProductTabsComponentProp
         }
     }
 
+    handleResize(entries: ResizeObserverEntry[]): void {
+        if (this.tabContentRef.current) {
+            this.tabContentRef.current.style.height = `${entries[0].contentRect.height }px`;
+        }
+
+        if (this.activeTabContentRef.current) {
+            this.activeTabContentRef.current.classList.add('fadeIn');
```

Comment:
instead of modifying the DOM element directly, better to utilize a `mods` prop. something like: `mods={ { isFadeIn } }`